src/Modules/OrchardCore.Commerce.Tax/Views/TaxRateSettings_Edit.cshtml

```diff
@@ -117,6 +129,8 @@
             "destinationPostalCode": null,
             "destinationRegion": null,
             "taxCode": null,
+            "vatNumber": null,
+            "isCorporation": @((int)MatchTaxRates.Ignored),
```

Comment:
`isCorporation` also should've been set to the default value. I made the change myself to avoid another round just for this small thing.